codec/src/main/java/io/netty/handler/codec/EmptyConvertibleHeaders.java

```diff
@@ -55,13 +55,9 @@ public ConvertedType getAndRemoveAndConvert(UnconvertedType name, ConvertedType
     }
 
     @Override
-    public List<Entry<ConvertedType, ConvertedType>> entriesConverted() {
-        return Collections.emptyList();
-    }
-
-    @Override
     public Iterator<Entry<ConvertedType, ConvertedType>> iteratorConverted() {
-        return entriesConverted().iterator();
+        List<Entry<ConvertedType, ConvertedType>> empty = Collections.emptyList();
```

Comment:
Not that it matters, but Collections.emptyIterator() and emptyListIterator() are available.